python/buildDcrCoadd.py

```diff
@@ -357,26 +365,31 @@ def build_model(self, verbose=True, max_iter=10, min_iter=None, clamp=None,
             initial_solution = self._model_spatial_filter(self.model, spatial_filter_use)
             if refine_max_iter is None:
                 refine_max_iter = max_iter*2
+            if refine_convergence_threshold is None:
+                refine_convergence_threshold = convergence_threshold/10.
```

Comment:
Any reasons for the magic numbers here other than just making the refined threshold an order of magnitude finer than the convergence_threshold?